Source/Scene/ModelInstanceCollection.js

```diff
@@ -72,6 +72,7 @@ define([
      * @param {Boolean} [options.asynchronous=true] Determines if model WebGL resource creation will be spread out over several frames or block until completion once all glTF files are loaded.
      * @param {Boolean} [options.debugShowBoundingVolume=false] For debugging only. Draws the bounding sphere for the collection.
      * @param {Boolean} [options.debugWireframe=false] For debugging only. Draws the instances in wireframe.
+     * @param {Boolean} [options.incrementallyLoadTextures=true] Determine if textures may continue to stream in after the model is loaded.
```

Comment:
Hey Sean, so I did make this change locally while I'm in the incrementallyLoadTextures branch. Would I just commit and push this change like last time?